adapters/base-quarkus/pom.xml

```diff
@@ -48,6 +48,10 @@
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-adapter-base-quarkus</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.eclipse.hono</groupId>
       <artifactId>hono-service-base-quarkus</artifactId>
```

Comment:
Wouldn't it be enough to have just `hono-adapter-base-quarkus` here?

Will the `hono-adapters-quarkus` module defined here ("Defines common dependencies for Quarkus-based protocol adapter implementations") then be become obsolete? It's a bit confusing to have both that module and the new `hono-adapter-base-quarkus` ("Base classes for implementing Hono protocol adapters using Quarkus.").
Same goes for the corresponding spring based modules.

Reply:
> Wouldn't it be enough to have just hono-adapter-base-quarkus here?

see other comment

> Will the hono-adapters-quarkus module defined here ("Defines common dependencies for Quarkus-based protocol adapter implementations") then be become obsolete? It's a bit confusing to have both that module and the new hono-adapter-base-quarkus ("Base classes for implementing Hono protocol adapters using Quarkus.").
Same goes for the corresponding spring based modules.

I don't think I get your point here.

Reply:
I was thinking of a module hierarchy with e.g. "service-base"->"adapter-base"->"adapter-spring"->"adapter-mqtt-spring" (with parent relationships). But I guess this would need more changes and would probably be less flexible. Maybe could be something to consider when we don't have to support spring anymore and only use quarkus.